backend/services/inventory/tests/tests/test_api_internal.py

```diff
@@ -88,28 +112,30 @@ def test_create_twice_ok(
         inventory_attributes.append(new_attr)
 
         # insert 'the same' device
-        _, r = internal_client.create_device(device_id=devid, attributes=new_attrs)
+        r = internal_client.initialize_device_with_http_info(
+            tenant_id="", device_new=oas.DeviceNew(id=devid, attributes=new_attrs)
+        )
         assert r.status_code == 201
 
         # verify update
-        dev = management_client.getDevice(devid)
+        dev = management_client.get_device_inventory(devid)
 
         self._verify_inventory(inventory_attributes, dev.attributes)
 
     def _verify_inventory(self, expected, inventory):
         # Filter only attributes within the inventory scope
         expected_inventory = list(filter(lambda a: a.scope == "inventory", expected))
         inventory = list(filter(lambda a: a.scope == "inventory", inventory))
-        assert len(inventory) == len(
-            expected_inventory
-        ), "expected: %s / actual: %s" % (inventory, expected_inventory)
+        assert len(inventory) == len(expected_inventory), (
+            "expected: %s / actual: %s" % (inventory, expected_inventory)
+        )
```

Comment:
you did run black formatter on it, right?

Reply:
Formatted using `black 25.1.0`